rfd/0010-api.md

```diff
@@ -147,17 +149,17 @@ Go module versions can automatically be pushed with git tags. Since the api modu
 
 Since Teleport has a major version above 2, the api must follow the [v2+ Go Module release guide](https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher) by appending a major version suffix to the import path. The import path for each major version `X` will look like `github.com/gravitational/teleport/api/vX`.
 
-This import path needs to be updated within the entire teleport repository. This can be done before each major release. This import path change is likely to cause easy to fix merge conflicts in backport PRs that are a full major version back.
+This import path needs to be updated within the entire teleport repository. This can be done before each major release. This import path change is likely to cause minor merge conflicts in backport PRs that are a full major version back.
 
-Additionally, if the git tag and import suffix aren't set up, the api will be versioned as v0.0.0-[random-string]. Getting the latest version with `go get github.com/teleport/api` will get the most recent version in terms of time rather than semver/teleport version, confusing and not user friendly. For this reason it's important to start tagging the version as soon as possible.
+Additionally, if the git tag and import suffix aren't set up, the api will be versioned as v0.0.0-[random-string]. Getting the latest version with `go get github.com/gravitational/teleport/api` will get the most recent version in terms of time rather than semver/teleport version, confusing and not user friendly. For this reason it's important to start tagging the version and add the `vX` suffix as soon as possible.
```

Comment:
So, for example, the command `go get github.com/gravitational/teleport/api/v7@v9.1.0` would mean "get me the Teleport 9.1 implementation of API version 7"? 

(Just an example, not that such a thing has to exist given our backwards compatibility rules)

Reply:
Actually I believe Go will return an error on that call. The major version suffix and the major version must line up, so the version suffix would have to be updated every major release.